README.md

```diff
@@ -28,7 +28,7 @@ powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://d
 
 For scripts, please see [Detect Scripts](https://github.com/synopsys-sig/synopsys-detect-scripts).
 
-For AirGap, please use our [Artifactory](https://sig-repo.synopsys.com/webapp/#/artifacts/browse/tree/General/bds-integrations-release/com/synopsys/integration/synopsys-detect).
+For AirGap, please use our [Artifact Repository](https://repo.blackduck.com/webapp/#/artifacts/browse/tree/General/bds-integrations-release/com/blackduck/integration/detect).
```

Comment:
@gopannabd This /webapp endpoint is not available anymore, this should be simply be https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect/

Reply:
I will correct it, thanks!